python/tk_multi_loader/loader_action_manager.py

```diff
@@ -96,8 +96,8 @@ def _get_actions_for_publish(self, sg_data, ui_area):
                                                         sg_publish_data=sg_data,
                                                         actions=actions,
                                                         ui_area=ui_area_str)
-        except Exception:
-            self._app.log_exception("Could not execute generate_actions hook.")
+        except Exception, e:
+            self._app.log_exception("Could not execute generate_actions hook: %s" % e)
```

Comment:
Heads up that this will include the exception message twice.

The following code:
```python
e = sgtk.platform.current_engine()

try:
    print "foo" + xyz
except Exception, ex:
    e.log_exception("Could not execute actions hook: %s" % ex)
```

Gives the following output:
```
# Error: Shotgun: tk-maya: Could not execute actions hook: name 'xyz' is not defined
Traceback (most recent call last):
  File "<maya console>", line 8, in <module>
NameError: name 'xyz' is not defined # 
```

The original code seems better to me:

```
e = sgtk.platform.current_engine()
try:
    print "foo" + xyz
except Exception:
    e.log_exception("Could not execute actions hook")
    
<Sgtk Engine 0x15be7d810: tk-maya, env: site>
# Error: Shotgun: tk-maya: Could not execute actions hook
Traceback (most recent call last):
  File "<maya console>", line 8, in <module>
NameError: name 'xyz' is not defined # 
```